sdk/identity/identity/src/msal/nodeFlows/nodeCommon.ts

```diff
@@ -94,7 +94,11 @@ export abstract class MsalNode extends MsalBaseUtilities implements MsalFlow {
       this.createCachePlugin = () => persistenceProvider!(options.tokenCachePersistenceOptions);
     } else if (options.tokenCachePersistenceOptions?.enabled) {
       throw new Error(
-        "Persistent token caching was requested, but no persistence provider was configured (do you need to use the `@azure/identity-cache-persistence` package?)"
+        [
+          "Persistent token caching was requested, but no persistence provider was configured.",
+          "You must install the identity-cache-persistence plugin package (`npm install --save @azure/identity-cache-persistence`)",
+          "and enable it by calling `useIdentityPlugin(cachePersistencePlugin)` before using tokenCachePersistenceOptions."
+        ].join(" ")
```

Comment:
nit: Seems wasteful to break the string into an array and join it. We can use string literals and new lines to make it as readable.